spec/features/user_can_see_a_book_spec.rb

```diff
@@ -0,0 +1,43 @@
+require 'rails_helper'
+
+describe "User sees a book page" do
+
+  before do
+    book = Book.create!(id: 1, title: "Throne of Glass")
+    User.create!(id: 1, name: "User 1")
+    User.create!(id: 2, name: "User 2")
+    User.create!(id: 3, name: "User 3")
+    Review.create!(rating: 1, content: "this book sucks", user_id: 1, book_id: 1)
+    Review.create!(rating: 3, content: "meh", user_id: 2, book_id: 1)
```

Comment:
as a shortcut, can refer to user_id just as user and same with book. rails knows that it should expect an id so it is implicit